examples/organization/variables.tf

```diff
@@ -35,6 +41,12 @@ variable "blocked_users" {
   ]
 }
 
+variable "create_all_members_team" {
+  description = "Whether or not to create a team that contains all users of the organization. The teams name can be set through the all_members_team_name"
+  type        = bool
+  default     = true
+}
+
```

Comment:
why do we need a second trigger? wouldn't it be enough to just create the team if a team name is set?

Reply:
resolved in 3b1d9e1